package.json

```diff
@@ -33,11 +33,13 @@
     "@uniswap/sdk-core": "3.0.1",
     "@uniswap/v3-sdk": "3.8.2",
     "@walletconnect/web3-provider": "1.8.0",
+    "@web3-onboard/core": "^2.6.0",
```

Comment:
nit: you could further break this stack up. one PR for the install, and yarn lock, the other for on board.

That way if there's something blocking onboard stuff you could still merge the dep updates on their own.

The rule I use is if and can say `and` it could probably be two different PRs.

This stack looks siiiiiick. Wooooo 🥲🎉

Reply:
Thanks for the heads up @benschac - really liking graphite so far! will do this as soon as I work out how to traverse the stack ;)

Reply:
Talked to @benschac and decided not to do this for now as it is feeling too confusing/risky